package.json

```diff
@@ -31,6 +31,7 @@
     "@typescript-eslint/parser": "^4.16.1",
     "@web/dev-server": "^0.1.1",
     "@web/test-runner": "^0.11.5",
+    "codecov": "^3.8.1",
```

Comment:
When you run things with `npx`, you don't have to add it to your dependencies

Reply:
Ah right, I believe I missed this when I switched from npm to npx. Thanks for pointing it out.